programs/sbf/rust/log_data/src/lib.rs

```diff
@@ -1,11 +1,12 @@
 //! Example Rust-based SBF program that uses sol_log_data syscall
 
-use solana_program::{
-    account_info::AccountInfo, entrypoint::ProgramResult, log::sol_log_data,
-    program::set_return_data, pubkey::Pubkey,
+use {
+    solana_account_info::AccountInfo, solana_program::program::set_return_data,
+    solana_program_error::ProgramResult, solana_pubkey::Pubkey,
+    solana_sysvar::program_stubs::sol_log_data,
```

Comment:
This should keep using solana-program too, since the stub doesn't use the syscall in solana builds